src/main/resources/META-INF/plexus/components.xml

```diff
@@ -17,6 +22,7 @@
        <package>org.apache.maven.plugins:maven-jar-plugin:2.6:jar,${project.groupId}:${project.artifactId}:${project.version}:build</package>
        <pre-integration-test>${project.groupId}:${project.artifactId}:${project.version}:start</pre-integration-test>
        <integration-test>org.apache.maven.plugins:maven-failsafe-plugin:2.19.1:integration-test</integration-test>
+       <!-- post-integration-test is opportunity for code coverage, such as jacoco, to collect statistics from running container -->
        <post-integration-test></post-integration-test>
        <verify>${project.groupId}:${project.artifactId}:${project.version}:stop,org.apache.maven.plugins:maven-failsafe-plugin:2.19.1:verify</verify>
```

Comment:
Ok, just checked it: any extra post-integration binding goes after the default one (strange behaviour btw, would expect for `pre` that custom executions goes after the default and for `post` before).

But if you need a customization you can still bind e.g. jacoco to the `integration-test` phase, which then will called *after* the integration test but before the `post-integration-phase`.

I adapted the lifecycle accordingly and also removed `docker:source` since this is duplicated in `docker:build`, too (adding to the build time)